app/scripts/metamask-controller.js

```diff
@@ -1920,6 +1943,41 @@ export default class MetamaskController extends EventEmitter {
     checkForMultipleVersionsRunning();
   }
 
+  getTxTrackingEventDependencyMap() {
```

Comment:
As mentioned in `transaction-metrics.ts`, this could become `getTransactionMetricsRequest`?

Reply:
Done in https://github.com/MetaMask/metamask-extension/commit/3e444bc285691f81624079dbdf6e1c329ed9eead